aspnetcore/fundamentals/aot/request-delegate-generator/diagnostics/rdg007.md

```diff
@@ -20,34 +20,19 @@ This diagnostic is emitted by the Request Delegate Generator when an endpoint co
 
 ### Rule description
 
-Types that are used for surrogate binding via the `AsParameters` attribute must contain a public parameterized constructor where all parameters to the constructor match the public properties declared on the type. The `TodoRequest` type will produce this diagnostic because there is no matching constructor parameter for the `Project` property.
+Types that are used for surrogate binding via the `AsParameters` attribute must contain a public parameterized constructor where all parameters to the constructor match the public properties declared on the type. The `TodoRequest` type will produce this diagnostic because there is no public constructor.
```

Comment:
I can revert the text changes if needed, but I already updated them to match the rule.